config/config.exs

```diff
@@ -7,6 +7,9 @@
 # General application configuration
 use Mix.Config
 
+config :skate,
+  gtfs_url: "https://cdn.mbta.com/MBTA_GTFS.zip"
```

Comment:
I know you are still working on this, so I'll wait on a full review, but this one line jumped out at me. Is this something that should be configured outside of the code base? Maybe we can discuss using a `.env` file or ENV vars or some other such solution, I'm not too particular.

Reply:
:+1: that's also one of the things called out in the ticket. Should we do a grooming next week to make sure everyone's on the same page w/ these tickets?